src/cli/cmds/new.ts

```diff
@@ -111,12 +111,18 @@ function renderParams(params: any) {
 
 function processDefault(value: string) {
   const basedir = path.basename(process.cwd());
-  const userEmail = execOrUndefined('git config --get --global user.email') ?? 'user@domain.com';
+  const userEmail = execOrUndefined('git config --get --global --include user.email') ?? 'user@domain.com';
```

Comment:
Is `--include` the same as `--includes`? (https://git-scm.com/docs/git-config#Documentation/git-config.txt---no-includes)

Reply:
Yes, sounds like this should be `--includes`:

```suggestion
  const userEmail = execOrUndefined('git config --get --global --includes user.email') ?? 'user@domain.com';
```

Reply:
Done, the interesting thing is: It works also